pyk/src/pyk/prelude/kint.py

```diff
@@ -13,24 +13,323 @@
 
 
 def intToken(i: int) -> KToken:  # noqa: N802
+    r"""Instantiate the KAST term ```#token`(i, "Int")``.
+
+    Args:
+        i: The left operand.
+
+    Returns:
+        The KAST term ```#token`(i, "Int")``.
+    """
```

Comment:
`#token` is not escaped in textual KAST:

```suggestion
    r"""Instantiate the KAST term ``#token(i, "Int")``.

    Args:
        i: The integer literal.

    Returns:
        The KAST term ``#token(i, "Int")``.
    """
```